packages/devtools-launchpad/package.json

```diff
@@ -77,7 +77,9 @@
     "node-static": "^0.7.7",
     "postcss": "^6.0.6",
     "postcss-bidirection": "^2.4.0",
+    "postcss-class-namespace": "^0.1.0",
```

Comment:
https://github.com/jasonLaster/postcss-class-namespace

might need to support some extra rules: like root and moz-selection... but it learned pretty quickly